docker/node/config/config-node.toml

```diff
@@ -4,6 +4,7 @@
 # WebSocket server bind address
 # type:string,ip
 address = "::ffff:0.0.0.0"
+enable = true
```

Comment:
Thanks!
`enable` should probably have the corresponding comments above it:
https://github.com/MacroChip/nano-node/blob/patch-1/nano/node/websocketconfig.cpp#L14

Reply:
```toml
# WebSocket server bind address.
# type:string,ip
address = "::ffff:0.0.0.0"

# Enable or disable WebSocket server.
# type:bool
enable = true
```

Reply:
Makes sense. Thanks